_init/README_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+# <orb-name> Orb [![CircleCI Build Status](https://circleci.com/gh/<org-name>/<repo-name>.svg?style=shield "CircleCI Build Status")](https://circleci.com/gh/<org-name>/<repo-name>) [![CircleCI Orb Version](https://img.shields.io/badge/endpoint.svg?url=https://badges.circleci.io/orb/<orb-namespace>/<orb-name>)](https://circleci.com/orbs/registry/orb/<orb-namespace>/<orb-name>) [![GitHub License](https://img.shields.io/badge/license-MIT-lightgrey.svg)](https://raw.githubusercontent.com/<org-name>/<repo-name>/master/LICENSE) [![CircleCI Community](https://img.shields.io/badge/community-CircleCI%20Discuss-343434.svg)](https://discuss.circleci.com/c/ecosystem/orbs)
+
+A description of your orb. Utilize this orb to easily add X to your CI/CD pipeline.
+
+TODO:
+When ready, you must _manually_ publish the first 1.0.0 version of your orb to initialize the production pipeline. After the first manual push you will no longer need to do this.
+
+1. When 1.0.0 is ready, create a PR to `master` with the commit message `1.0.0 [semver:skip]`.
```

Comment:
I'm curious about the need for manually publishing the first 1.0.0 version via CLI because I tested making a PR from `Alpha` to `master` with [semver:major] and the orb was able to be promoted via the CI pipeline, even though it was the first production release

Reply:
Sweet! 🎉 This was carried over from the old instructions (which I thought I remembered needing at the time), but I knew I wanted to test it again but was attempting to avoid creating a production Orb until the next time I needed one. Very happy to hear that it appears to work automatically though. I'll remove this and we are all set for a while!